pkg/scheduler/factory/factory.go

```diff
@@ -574,6 +574,7 @@ func (c *configFactory) getPluginArgs() (*PluginFactoryArgs, error) {
 		NodeLister:                     &nodeLister{c.nodeLister},
 		PDBLister:                      c.pdbLister,
 		NodeInfo:                       c.schedulerCache,
+		CSINodeInfo:                    c.schedulerCache,
```

Comment:
Initially I re-used the `NodeInfo` interface above by adding a new `GetCSINodeInfo()` method to it. However, the patch got fairly bigger because the `NodeInfo` interface is used in many places in k8s.